tests/sync_filter_test.go

```diff
@@ -0,0 +1,36 @@
+package tests
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+func TestSyncFilter(t *testing.T) {
+	deployment := Deploy(t, "sync_filter", b.BlueprintAlice)
```

Comment:
Please remove `"sync_filter"` from this as it is no longer required as of https://github.com/matrix-org/complement/pull/113